code/procedures/add-mods-to-specs.py

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env python
+
+
+def add_name_fixing(path):
+    import datalad.support.json_py as json_py
+
+    spec = [r for r in json_py.load_stream(path)]
+
+    # Note: We append the procedure to dicomseries:all, since we do not
+    # technically act upon a particular series. This is because the procedure
+    # concerns the outcome of the conversion, not the raw data. The file
+    # selection has to be done within the procedure and can't be controlled by
+    # the spec or hirni-spec2bids ATM.
+    for snippet in spec:
+        if snippet['type'] == 'dicomseries:all':
+            snippet['procedures'].append(
+                {
+                    'procedure-name': {'value': 'change-dwi-run-to-acq_fix_all',
+                                       'approved': True},
+                    'procedure-name': {'value': 'fieldmaps-to-phase-or-magnitude_fix_all',
+                                       'approved': True},
+                    'on-anonymize': {'value': False,
+                                     'approved': True},
+                }
+            )
```

Comment:
Nope. If you want to add two procedures, then it's two dictionaries to be appended:
```python
snippet['procedures'].append(
    {
        'procedure-name': {'value': 'change-dwi-run-to-acq_fix_all',
                                     'approved': True},
        'on-anonymize': {'value': False,
                                 'approved': True},
    }
)
snippet['procedures'].append(
     {
        'procedure-name': {'value': 'fieldmaps-to-phase-or-magnitude_fix_all',
                                     'approved': True},
        'on-anonymize': {'value': False,
                                 'approved': True},
    }
)
```

Alternatively, extend the list rather than appending twice, like: snippet['procedures'].extend( [{...}, {...}] )
You can't have a dictionary where the same key ("procedure-name") appears twice.